datameta/static/js/admin.js

```diff
@@ -184,12 +185,398 @@ DataMeta.admin.reload = function() {
             .then(function (json) {
                 DataMeta.admin.reload_requests(json.reg_requests, json.groups);
                 DataMeta.admin.subnav();
+                DataMeta.admin.rebuildUserTable(json.users);
+                DataMeta.admin.rebuildGroupTable(json.groups);
+                DataMeta.admin.groups = json.groups;
             })
             .catch((error) => {
                 console.log(error);
             });
 }
 
+//Rebuilds the user table, based on the Data fetched from the API
+DataMeta.admin.rebuildUserTable = function(users) {
+    var t = $('#table_users').DataTable()
+    t.clear();
+    t.rows.add(users);
+    t.draw();   
+}
+
+//Initializes the user table
+DataMeta.admin.initUserTable = function() {
+    $('#table_users').DataTable({
+        destroy: true, //Destroys the table, in case it already exist
+        rowId: 'uuid',
+        order: [[1, "asc"]],
+        paging : true,
+        searching: false,
+        columns: [
+            { title: "User ID", data: "id"},
+            { title: "Name", data: "fullname", render:function(data) {
+                return '<button type="button" class="py-0 px-1 btn btn-sm enabled" onclick="changeUserName(event);" data="' + data + '">' + data + ' <i class="bi bi-pencil-square"></i></button>';
+            }},
+            { title: "Email Adress", data: "email" },
```

Comment:
```suggestion
            { title: "Email Address", data: "email" },
```
'address' is still misspelled 😉

Reply:
Ah it's not the same location. Must have overlooked that one before.